CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Resolver
 
+### Metroid Prime
+
+- Added: FAQ Entry about non-Superheated rooms
```

Comment:
wrong place. youre now suggesting that the seeds entry from below is a prime thing